tests/test_types.py

```diff
@@ -1864,3 +1864,37 @@ def test_dumping_passes_context_to_inner_type_dump(self):
         context = object()
         DumpOnly(inner_type).dump('foo', context)
         assert inner_type.dump_context == context
+
+class TestTransform:
+    def test_transform_simple_transform(self):
+        inner_type = SpyType()
+        transform = Transform(inner_type,
+                              load_modifier=lambda foo: 'foo',
```

Comment:
This test does not check which value will be passed to modifier. Also, it does not check which value will be passed to inner type. All this checks should all be separate tests.